web/themes/custom/cloudy/templates/block/block--system-branding-block.html.twig

```diff
@@ -12,7 +12,7 @@
  */
 #}
 {% if site_logo or site_name %}
-  <a href="{{ path('<front>') }}" alt="{{ 'Go to the official City of Portland, Oregon website homepage.'|t }}" title="{{ 'Go to the official City of Portland, Oregon website homepage.'|t }}" rel="home" class="navbar-brand d-flex align-items-center">
+  <a href="{{ path('<front>') }}" aria-label="{{ 'Go to the official City of Portland, Oregon website homepage.'|t }}" title="{{ 'Go to the official City of Portland, Oregon website homepage.'|t }}" rel="home" class="navbar-brand d-flex align-items-center">
     {% if site_logo %}
       <img src="{{ site_logo }}" alt="{{ "Image of the official City of Portland seal. The image depicts Portlandia holding her trident backdropped by mountain and river, accompanied by the text 'City of Portland, Oregon 1851'"|t }}" class="cloudy-logo img-fluid" />
```

Comment:
Can we drop "Image of the" from this line? I believe this is what Lisa was reporting?